docs/Sonarr/Sonarr-Release-Profile-RegEx.md

```diff
@@ -224,78 +136,7 @@ Add this to your `Must not contain (2)`
 
 ------
 
-### P2P Groups + Repack/Proper
-
-Add this to your `Preferred (3)` with a score of [180]
-The reason why this one get's such a high score is because it's the only quality scene group that exist (till now) and scene groups don't add the [Release Sources (Streaming Service)](#release-sources-streaming-service) to their release name so they don't get the extra point of the release source regex.
```

Comment:
This changes. The paragraph now prints before `Add this to your `Preferred (3)` with a score of [180]`.

That OK?